Formula/unbound.rb

```diff
@@ -13,11 +13,13 @@ class Unbound < Formula
 
   depends_on "openssl"
   depends_on "libevent"
+  depends_on "expat"
```

Comment:
I think if we adopt the code block I proposed that'll get it working for everyone without making 'em install the CLT to get there.